test/built-ins/Iterator/prototype/Symbol.toStringTag/prop-desc.js

```diff
@@ -1,20 +1,18 @@
 // Copyright (C) 2023 Michael Ficarra. All rights reserved.
 // This code is governed by the BSD license found in the LICENSE file.
 /*---
-es6id: 25.1.2.1
+esid: sec-iteratorprototype-@@tostringtag
 description: Property descriptor
 info: |
-    ES6 Section 17
-
-    Every other data property described in clauses 18 through 26 and in Annex
-    B.2 has the attributes { [[Writable]]: true, [[Enumerable]]: false,
-    [[Configurable]]: true } unless otherwise specified.
+  `Iterator.prototype[@@toStringTag]` is an accessor property with attributes { [[Enumerable]]: *false*, [[Configurable]]: *true* }
 features: [iterator-helpers]
 includes: [propertyHelper.js]
 ---*/
-verifyProperty(Iterator.prototype, Symbol.toStringTag, {
-  value: 'Iterator',
-  writable: true,
-  enumerable: false,
-  configurable: true,
-});
+verifyConfigurable(Iterator.prototype, Symbol.toStringTag);
```

Comment:
This test and the corresponding one for `constructor` property are broken. `verifyConfigurable()` calls `isConfigurable()` that [deletes](https://github.com/tc39/test262/blob/main/harness/propertyHelper.js#L122) the property. So, the subsequent parts of this test fail because the property is not there anymore.

If you want to use `verifyConfigurable()` then it should be the last statement of the test.